docs/application/native/guides/multimedia/image-edit.md

```diff
@@ -300,19 +418,16 @@ To decode a JPEG, PNG, GIF, or BMP image:
 
    > **Note**
    >
-   > - Because of decoder limitations, color space setting and JPEG downscaling are only supported for JPEG images.
-   > - The default color space is `IMAGE_UTIL_COLORSPACE_RGBA8888`. PNG, GIF and BMP images are decoded to `IMAGE_UTIL_COLORSPACE_RGBA8888`.
+   > Due to the decoder limitations, the color space setting is only supported for decoding the JPEG images. The default color space is `IMAGE_UTIL_COLORSPACE_RGBA8888`. PNG, GIF, and BMP images are encoded with `IMAGE_UTIL_COLORSPACE_RGBA8888`.
```

Comment:
... PNG, GIF, and BMP images are **encoded** with...

According to the note (and @jiyong2's comment) as PNG, GIF and BMP are all encoded/decoded with IMAGE_UTIL_COLORSPACE_RGBA8888, the default color space.
However would you, please check whether the "emcoded" is correct in that location?

The note is in decoding section.

Reply:
Hello, @jiyong2. Would you please check this comment again?

Reply:
I'm sorry for the delay. The 'encoded with' is incorrect. It should be changed 'decoded with'.
I will replace the error.